source/extensions/filters/common/ext_proc/grpc_client_impl.h

```diff
@@ -226,7 +226,9 @@ class ProcessorClientImpl : public ProcessorClient<RequestType, ResponseType> {
         Http::StreamFilterSidestreamWatermarkCallbacks& sidestream_watermark_callbacks) override {
     auto client_or_error =
         client_manager_.getOrCreateRawAsyncClientWithHashKey(config_with_hash_key, scope_, true);
-    THROW_IF_NOT_OK_REF(client_or_error.status());
+    if (!client_or_error.status().ok()) {
+      return nullptr;
```

Comment:
are there any logging/stats tracked this error case?

I feel it is good to have a logging line here?

Reply:
+1. Let's add a LOG_EVERY_N here.

Reply:
Done

Reply:
probably too spammy if there are a lot of request, let's do log_every_N_sec(10)

Reply:
This failure is very rare, and if it happens, we should just log it.  BTW, there is no log_every_N_sec() in OSS.

Reply:
it will log too much when the config is in trouble, right now linear to QPS, I don't think that's worth that much.

Reply:
could you use the ENVOY_LOG_PERIODIC_MISC macro, weird naming.